docs/en/setup/service-agent/java-agent/How-to-enable-kafka-reporter.md

```diff
@@ -0,0 +1,16 @@
+# How to enable Kafka Reporter
```

Comment:
This doc has not been linked from `java-agent/README.md`. It should be a separate section after `## Advanced Features`

```markdown
## Advanced Reporters
* Use Kafka to transport the traces, JVM metrics, instance properties, and profiled snapshots to the backend. Read the [doc](How-to-enable-kafka-reporter.md) for more details.
```